plugin-core/resources/bzleclipse_aspect.bzl

```diff
@@ -123,7 +147,7 @@ def _aspect_impl(target, ctx):
         hasDepAttr = True
 
   hasJavaAttr = False
-  if hasattr(target, "java"):
+  if JavaInfo in target:
```

Comment:
java_grpc_library does not use name 'java' for the JavaInfo provider, so had to change this 'if' statement here.